app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+
+from .errors import NotVaccinatedError
+from .errors import OutdatedVaccineError
+from .errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" in visitor:
+            if ("expiration_date" in visitor["vaccine"]
+                    and datetime.date.today()
+                    <= visitor["vaccine"]["expiration_date"]):
+                if visitor["wearing_a_mask"]:
+                    return f"Welcome to {self.name}"
+                else:
+                    raise NotWearingMaskError
```

Comment:
The checklist advises using descriptive error messages when raising exceptions. Consider providing a message for NotWearingMaskError, e.g., raise NotWearingMaskError('Visitor is not wearing a mask.').